yarn-error.log

```diff
@@ -0,0 +1,12438 @@
+Arguments: 
+  /usr/local/bin/node /usr/share/yarn/bin/yarn.js check-linting
+
+PATH: 
```

Comment:
This file seems unintentional

Reply:
Yes it is unintentional, Let me remove it